data/imagery.json

```diff
@@ -10515,6 +10515,7 @@
             0,
             16
         ],
+        "overzoom": false,
```

Comment:
This will get overwritten by `make imagery`. Can you add some code to `data/update_imagery.js` that adds `imagery: false` to the entry for the locator overlay as a special case?